helm/templates/admission-webhooks/operator-webhook-with-cert-manager.yaml

```diff
@@ -95,6 +95,66 @@ webhooks:
     - pods
   sideEffects: None
   timeoutSeconds: {{ .Values.admissionWebhooks.timeoutSeconds }}
+- admissionReviewVersions:
+  - v1
+  clientConfig:
+    service:
+      name: {{ template "amazon-cloudwatch-observability.webhookServiceName" . }}
+      namespace: {{ .Release.Namespace }}
+      path: /mutate-v1-namespace
+  failurePolicy: {{ .Values.admissionWebhooks.pods.failurePolicy }}
+  name: mnamespace.kb.io
+    {{- if .Values.admissionWebhooks.namespaceSelector }}
+  namespaceSelector:
+    {{- toYaml .Values.admissionWebhooks.namespaceSelector | nindent 6 }}
+    {{- end }}
+    {{- if .Values.admissionWebhooks.objectSelector }}
+  objectSelector:
+    {{- toYaml .Values.admissionWebhooks.objectSelector | nindent 6 }}
+    {{- end }}
+  rules:
+  - apiGroups:
+    - ""
```

Comment:
`"core"`? ([ref](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.29/#namespace-v1-core))

Reply:
I picked this up from a manifest that @jefchien had tested in an earlier PR.
But I also just checked on a cluster and namespaces do not actually have an apiGroup. Im guessing empty defaults to core.
```
kubectl api-resources -o wide | grep "deployments\|namespaces"
namespaces                        ns                       v1                                     false        Namespace                        create,delete,get,list,patch,update,watch                    
deployments                       deploy                   apps/v1                                true         Deployment                       create,delete,deletecollection,get,list,patch,update,watch   all
```

Reply:
Right I see the same being referenced in a couple of places - that "" means `core`.

Also, in our `ClusterRole` [here](https://github.com/aws/amazon-cloudwatch-agent-operator/pull/85/files#diff-9cf6c12a0631332ed0ce9158f1c5c9b6fe11b36206c87e73f3347309a827d866R12) or in CI samples [here](https://github.com/aws-samples/amazon-cloudwatch-container-insights/blob/main/k8s-deployment-manifest-templates/deployment-mode/daemonset/container-insights-monitoring/cwagent/cwagent-serviceaccount.yaml#L14), we dont see to use `core`.

It shouldnt hurt to explicitly say `core`, so I dont mind testing and updating accordingly.

Reply:
I tested this real quick and if I set this apiGroup to `core`, the webhook actually doesnt work as expected.

Reply:
Thanks for checking!  Seems like a problem that there's no way to specify core only, but oh well, in practice it's not really an issue since we restrict to the namespaces resource.